BusFilter/Driver.c

```diff
@@ -160,7 +160,7 @@ BusFilterQueryID(WDFOBJECT BffDevice, PIRP Irp)
                 oldIDs += wcslen(oldIDs)+1;
             }
             *newIDs = 0;
-            ExFreePool((PVOID)Irp->IoStatus.Information);
+            ExFreePoolWithTag((PVOID)Irp->IoStatus.Information, 0);
```

Comment:
Since Line 145 uses 'tFFB' as the tag for ExAllocatePoolWithTag, shouldn't the second parameter of ExFreePoolWithTag be 'tFFB'?

Reply:
No, because the "owner" of that block of memory is the upper driver, ans since we don't know their tag, the default of 0 is recommended.